apis/condition_set.go

```diff
@@ -52,6 +52,9 @@ type ConditionManager interface {
 	// set to true.
```

Comment:
Might consider updating this comment with something about that if you need additional details (for propagating the status in case of dependencies, to look at GetTopLevelCondition), and if you only need to know if something is indeed happy to use this method.